pkg/image/imageversion.go

```diff
@@ -87,7 +87,7 @@ func (c *ConformanceImageVersion) Set(str string) error {
 // Don't require the entire kubernetes.Interface to simplify the required test mocks
 func (c *ConformanceImageVersion) Get(client discovery.ServerVersionInterface, latestURL string) (registry, version string, returnErr error) {
 	switch *c {
-	case ConformanceImageVersionAuto:
+	case "", ConformanceImageVersionAuto:
```

Comment:
Consistent with how empty value should be treated like the default value in most cases.